opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew.adoc

```diff
@@ -1,6 +1,12 @@
 [[releasenotes-25]]
 == What's New in OpenNMS Horizon 25
 
+=== System Requirements
+
+* *Java 8 through 11*: OpenNMS Horizon 25 requires at least Java 8 as the runtime environment, as well as JDK 11.
+  To run Horizon 25, we recommend the most recent version of OpenJDK 11.
+* *PostgreSQL 10 or higher*: Horizon 25 requires Any supported version of PostgreSQL 10 or higher.
```

Comment:
I suspect that users will need some help in moving from PG 9.x to PG 10+. Can we give them any useful references?